regolith/schemas.py

```diff
@@ -3620,6 +3621,172 @@
+                "due_date": {"description": "kickoff meeting by date, yyyy-mm-dd.",
+                             "required": False,
+                             "anyof_type": ["date", "string"]},
+                "name": {"description": "name of meeting",
+                         "required": False,
+                         "type": "string"},
+                "objective": {"description": "goal of the meeting",
+                              "required": False,
+                              "type": "string"},
+                "audience": {"description": "rolls of audience members "
+                                            "e.g. 'pi', 'lead'",
+                             "required": False,
+                             "type": "list"},
+                "notes": {"description": "any notes about the kickoof",
+                          "required": False,
+                          "type": "list"},
+                "status": {"description": "proposed, started, finished, converged",
+                           "required": False,
+                           "type": "string",
+                           "eallowed": PROJECTA_STATUS}
+
+            }
+        },
+        "lead": {
+            "description": "id of projecta lead",
+            "required": False,
+            "type": "string"
+        },
+        "log_url": {
+            "description": "link to log of meetings",
+            "required": False,
+            "type": "string"
+        },
+        "milestones": {
+            "description": "smaller deliverables done by a certain date "
+                           "a series of milestones ends with the deliverable "
+                           "the projecta",
+            "required": False,
+            "type": "list",
+            "schema": {
+                "type": "dict",
+                "schema": {
+                    "due_date": {"description": "due date of milestone, yyyy-mm-dd",
+                                 "required": False,
+                                 "anyof_type": ["date", "string"]},
+                    "name": {"description": "what is the deliverable of milestone",
+                             "required": False,
+                             "type": "string"},
+                    "notes": {"description": "small, non-deliverable to-do's",
+                              "required": False,
+                              "type": "list"},
+                    "objective": {"description": "explains goal of the milestone",
+                                  "required": False,
+                                  "type": "string"},
+                    "audience": {"description": "rolls of audience members "
+                                                "e.g. 'pi', 'lead'",
+                                 "required": False,
+                                 "type": "list"},
+                    "status": {"description": "proposed, started, finished, converged",
```

Comment:
ouch.   use an fstring here and say f"status of the milestone from {MILESTONE_STATI}" sot the documentation stays up to date with the code as the code develops (e.g., if we decide to add a milestone status.....

Reply:
I didn't understand this comment completely, so I likely didn't implement it how you were envisioning.

Reply:
> 
> 
> I didn't understand this comment completely, so I likely didn't implement it how you were envisioning.

I tried to give a more complete description in the latest comment